docs/Configure-GenericAptRepo.md

```diff
@@ -0,0 +1,23 @@
+# How to configure a generic aptitude repo
+
+This is made on the latest ubuntu-xenial.
+
+## On the repo machine:
+
+Make a dir containg all debs you want to share, in this example it will be ~/deb
+```bash
+$ sudo apt-get install apache2 apt-transport-https dpkg-dev
+$ sudo cp -r ~/deb /var/www/html
+$ cd /var/www/html/deb
+$ dpkg-scanpackages -m . | sudo gzip -c > Packages.gz # this will be required to be run every time you add/remove a package
+```
+
+## On the client
+```bash
+$ sudo sh -c  'echo "deb [trusted=yes] http://<repo ip>/deb ./" >> /etc/apt/sources.list'
```

Comment:
what is the repo-ip?

repo_ip is the Ubuntu machine's IP which has the apt repository configured.